docs/best_practices/time_series.rst

```diff
@@ -145,3 +145,20 @@ chunk individually. This is especially important when writing NWBFiles that are
 
 Check functions: :py::meth:`~nwbinspector.checks.nwb_containers.check_large_dataset_compression`,
 :py::meth:`~nwbinspector.checks.nwb_containers.check_small_dataset_compression`
+
+
+
+.. _best_practice_time_series_data_is_not_none:
+
+Do Not Store Data as ``None``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Specific to the :pynwb-docs:`PyNWB API <>`, when writing data to a :nwb-schema:ref:`sec-TimeSeries` object, or any
+descendant, do not use the native Python type of ``None``, as it is not a
+:nwb-schema:ref:`supported value <common-attributes>` in the NWB schema.
```

Comment:
best practices should not be language-specific. How would this sentence apply to someone using MATLAB?

Reply:
Yes, this section is pretty specific to Python, since I have no idea how MATLAB handles situations like this; all I know is if the `data` is missing, and you read it using `PyNWB` (as the Inspector does), it shows up in the end as `None`.

I guess we could just refer to it as being 'missing' in the docs?